chart/cas-cif/.helmignore

```diff
@@ -21,3 +21,11 @@
 .idea/
 *.tmproj
 .vscode/
+
```

Comment:
Can we add the below to the .gitignore as well :)

Reply:
Added in 81e56976e87a31cf74ca2dd3c1d6d820efd2295e